src/foam/comics/DAOController.js

```diff
@@ -92,7 +92,7 @@ foam.CLASS({
       class: 'Boolean',
       name: 'exportCSVEnabled',
       documentation: 'True to enable export csv button',
-      value: false
+      value: true
```

Comment:
Undo this change or remove the `exportCSVEnabled` property completely.

Reply:
I'm just setting this property back to being false. 

But ... Did we want to get rid of it? Don't we want default true??

I'm going to make a ticket for the parser portion of this, and put this in as a note on TODO

Reply:
Yeah we want to get rid of it but doing so might break some things right now so it's best to leave this untoutched for now and not change anybody's behaviour just yet.

Having this `exportCSVEnabled` is more code for less functionality than simply having the `exportEnabled` flag. We should only need the `exportEnabled` and then let the user decide how they'd like to export. Also, having the `exportCSVEnabled` creates a bad precedent for having a separate flag for all of the output types. Before you know it, we'd have an `exportJSONEnabled`, `exportXMLEnabled`, etc. and there's not much value in that when we could have a single `exportEnabled` to rule them all.

Reply:
LOL OK got it. Makes sense. Thanks as always for explaining 👍 🥇

Reply:
https://nanopay.atlassian.net/browse/CPF-695